infarstructure/src/main/java/util/InfraUtil.java

```diff
@@ -0,0 +1,35 @@
+package util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.Gson;
+import com.google.gson.stream.JsonReader;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+
+public class InfraUtil {
+
+    // jsonString to JSON object
+    public static <T> T stringToObject(String json, Class<T> tClass) {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
You can have one `private final static ObjectMapper mapper = new ObjectMapper()`